sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/implementation/jackson/JsonNodeUtils.java

```diff
@@ -9,7 +9,10 @@
 import java.util.Map;
 import java.util.Set;
 
-abstract class JsonNodeUtils {
```

Comment:
refer: https://sonarcloud.io/project/issues?directories=sdk%2Fspring%2Fspring-cloud-azure-autoconfigure%2Fsrc%2Fmain%2Fjava%2Fcom%2Fazure%2Fspring%2Fcloud%2Fautoconfigure%2Faad%2Fimplementation%2Fjackson&resolved=false&types=CODE_SMELL&id=stliu_azure-sdk-for-java&open=AXtAG3K-00FAjxsg1BbB